.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug in this library
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+This form is for bug reports for this library only.
+For general, language-agnostic questions and troubleshooting, please ask/look for answers [here](https://groups.google.com/g/adwords-api).
+
+Please check existing issues to see if your bug has already been reported before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
+-->
+
+**Which version of the client library are you using?**
+
+**Which version of PHP are you using?**
+
+**Which operating system (Linux, Windows, ...) and version?**
+
+**Actions taken**
+Please provide a code snippet or steps to consistently reproduce the experienced behavior.
+[DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA IN YOUR SUBMITTED CODE]
+
+**Expected result**
+
+**Actual result**
+
+Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).
```

Comment:
@devchas should we add a link to the dedicated documentation we have for PHP?

Reply:
SGTM

Reply:
I added a link to the logging page. Please let me know if you want to suggest adding other links.

Reply:
Thank you both, it LGTM.